frontend/awx/access/credentials/CredentialForm.tsx

```diff
@@ -111,8 +121,22 @@ export function CreateCredential() {
         submitText={t('Create credential')}
         onSubmit={onSubmit}
         onCancel={() => navigate(-1)}
+        additionalActionText={isExternalCredential ? 'Test' : undefined}
```

Comment:
Do we need select this one to be translated?

Reply:
Good catch, thanks